parkings/importers/payment_zones.py

```diff
@@ -0,0 +1,96 @@
+import logging
+
+from django.db import transaction
+from lxml import etree
+from owslib.wfs import WebFeatureService
+
+from parkings.models import PaymentZone
+
+from .utils import get_polygons
+
+logger = logging.getLogger(__name__)
+
+
+class PaymentZoneImporter(object):
```

Comment:
Quickly looking this seems to have quite a lot of duplication with `ParkingAreaImporter`. I wonder if something should be done about it or will the importers just get overly complicated... Would like to hear @suutari-ai  opinion about this.

Reply:
I think you're right about the duplication. IMHO such duplication should be avoided and rather the importers should be refactored so that they can share as much of the code as possible.

Reply:
That is a very good point. Indeed this is almost the exact copy of `ParkingAreaImporter`. I will refactor to abstract the duplication. Thank you @japauliina , @suutari-ai .